### ISSUE TITLE
now alias - not down scaling previous deployment

### ISSUE THREAD

**Issue by jthegedus** [vercel/vercel#1324]
`now alias` successfully scales up the new deployment as per the scaling rule applied to my domain, great :+1:

My problem is that the default scaling rule of `min:0 max:1` is not applied to the previous instance. It remains with the scaling rules from the domain, which have a `min:3` so I'm constantly finding myself over the limit and paying for on-demand. With CD and deploying multiple times a day this is a problem.

I am specifying the deployment id and url in the alias command:

```
now alias set $NOW_DEPLOYMENT_ID $URL --team $NOW_TEAM --token $NOW_TOKEN
```

Interestingly, when I set `NOW_DEPLOYMENT_ID` to be the current aliased instance, it applies the default scaling rules to what is my production instance.

**now-cli version**: `now@12.0.0-canary.5`

**Comment by javivelasco** [comment]
Hi! Can you try with the latest canary? I can't reproduce the issue, maybe it's already fixed

**Comment by jthegedus** [comment]
This does seem to be fixed in `now@12.0.0-canary.23`. Thanks for trying to reproduce this :100:
